docs/scenarios/parametric_contributing.md

```diff
@@ -0,0 +1,70 @@
+
+If your usage does not require different parameter values, then [end-to-end system-tests](/docs/README.md) should be used as they will achieve the same level of behavior uniformity verification and test the feature on real world use cases, catching more issues.
+
+System-tests are **not** for testing internal or niche tracer behavior. Unit tests are a better fit for that case.
+
+## Getting set up
+
+We usually add new system tests when validating a new feature. This feature might not yet be implemented across all dd-trace libraries. If at least one library already supports the feature, you can verify your test by running it against that library.
+
+To begin we need to make sure system-tests run with a tracer that has implemented the feature being tested (published or on a branch).
+Follow [Binaries Documentation](../execute/binaries.md) for your particular tracer language to set this up.
+
+[Verify that you can run some parametric tests with your custom tracer](parametric.md#running-the-tests). Make sure some pass (no need to run the whole suite, you can stop the tests from running with `ctrl+c`). If you have an issue, checkout the [debugging section](parametric.md#debugging) to troubleshoot.
+
+## Writing the tests
+
+Now that we're all setup with a working test suite and a tracer with the implemented feature, we can begin writing the new tests.
+
+**MUST:** If you haven't yet, please acquaint yourself with [how system tests work](parametric.md#architecture-how-system-tests-work) before proceeding and reference it throughout this section.
+
+First take a look at the [currently existing tests](/tests/parametric) and see if what you're trying to test is similar and can use the same methods/endpoints, in many cases new endpoints do not need to be added.
+
+For a list of client methods that already exist, refer to `class APMLibrary` in the [_library_client.py](/utils/parametric/_library_client.py). If you're wondering what the methods do, you can take at look at the respective endpoints they're calling in that same file in `class APMLibraryClient`.
+
+The endpoints (where the actual tracer code runs) are defined in the Http Server implementations per tracer [listed here](parametric.md#http-server-implementations). Click on the one for your language to take a look at the endpoints. In some cases you may need to just slightly modify an endpoint rather than add a new one.
+
+### If you need to add additional endpoints to test your new feature
+
+*Note:* please refer to the [architecture section](parametric.md#architecture-how-system-tests-work) if you're confused throughout this process.
```

Comment:
We should link to this dashboard to show endpoints that exist but are not up to spec: https://feature-parity.us1.prod.dog/#/?runDateFilter=7d&feature=339

Reply:
I'm going to put this in parametric.md instead.  I don't think this should be in the main path of a first time contributor.